### ISSUE TITLE
One Name to Rule Them All (Standardize our Project Name)

### ISSUE THREAD

**Issue by rachelbaker** [WP-API/WP-API#850]
Our repository and GH organization name is "WP-API", which I know we are trying to move away from.  While trying to move away from WP-API, we haven't found a consistent alternative and it is very confusing.  

Can we find one preferred name for this project/plugin and use it uniformly throughout the project?   

Examples of our current variants:

1. WP REST API - title on the [documentation site](http://wp-api.org/).
2. REST API - content and sidebar menu heading on the [documentation site](http://wp-api.org/).
3. JSON REST API (WP-API) - main [README](https://github.com/WP-API/WP-API/blob/develop/README.md)  heading.
4. JSON REST API (WP API) - WordPress.org [plugin name](https://wordpress.org/plugins/json-rest-api/).
5. JSON REST API - plugin name seen from within the WordPress Dashboard.
6. JSON API - [Basic Auth](https://github.com/WP-API/Basic-Auth) repo heading

**Comment by rachelbaker** [comment]
My vote would be something like:

WP JSON API
or
WP REST API

I think we should keep the WP to give context to what the API is for (WordPress), and WP JSON REST API just looks too long.

**Comment by kosso** [comment]
Is WP-API/OAuth1 going to be part of WP-API/WP-API eventually?

**Comment by rachelbaker** [comment]
@kosso Merging the OAuth 1 plugin is on the Roadmap for our Core Integration: https://github.com/WP-API/WP-API/issues/571

**Comment by kadamwhite** [comment]
`wp plugin install WordPress-JavaScript-Object-Notation-Representational-State-Transfer-Application-Programming-Interface`

More seriously, I recall @nacin objecting to calling it explicitly a JSON API, because theoretically we could keep the infra and change the transport, but people do think of it that way: we get a lot of issues on this repo relating to name conflict with the existing JSON API plugins.

In this context, I would vote "WP REST API."

**Comment by danielbachhuber** [comment]
I like WP REST API.

**Comment by rmccue** [comment]
Previously discussed over on https://github.com/WP-API/WP-API.github.io/issues/10

I'm also +1 on WP REST API, and definitely -1 on anything involving "JSON" in the name. JSON isn't the fundamental technology with the API (it's merely one of possibly many serialisation formats), REST *is*, and it forms the core of how we design and build the API. :)

Let's make it so.

**Comment by rachelbaker** [comment]
http://media.tumblr.com/37579093b57e769ef1a3e58c299600ad/tumblr_inline_n4edjqIgaC1qzpq7e.gif

**Comment by nacin** [comment]
Yep.

I've probably called it the JSON API at one point or another, but I've definitely been primarily pushing REST API. I've just always looked to avoid WP API. :+1: for WP REST API.

Some evidence: I've tweeted a bunch of times about the "REST API" but only ever mentioned a possible future "JSON API" in 2012. I'm mainly stating this because I found this in the process, and thought some of you might chuckle: https://twitter.com/nacin/status/286690632511143936.
